goss-testing/scripts/postgresql_backups_check.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+print_results=0
+while getopts ph stack
+do
+    case "${stack}" in
+          p) print_results=1;;
+          h) echo "usage: postgres_backups_check.sh           # Only print 'PASS' upon success"
```

Comment:
The name of the script is postgresql_backups_check.sh

Reply:
fixed